plutus-ledger-api/src/v3/transaction.rs

```diff
@@ -26,26 +43,210 @@ use crate::{
     },
 };
 
-use super::ratio::Rational;
+use super::{
+    crypto::{ledger_bytes, Ed25519PubKeyHash, LedgerBytes},
+    ratio::Rational,
+};
+
+/////////////////////
+// TransactionHash //
+/////////////////////
+
+/// 32-bytes blake2b256 hash of a transaction body.
+///
+/// Also known as Transaction ID or `TxID`.
+/// Note: Plutus docs might incorrectly state that it uses SHA256.
+/// V3 TransactionHash uses a more efficient Plutus Data encoding
```

Comment:
```suggestion
/// V3 TransactionHash uses a more efficient Plutus Data encoding.
/// See <https://github.com/IntersectMBO/plutus/blob/1.38.0.0/plutus-ledger-api/src/PlutusLedgerApi/V3/Tx.hs#L37-L54> for details.
```

Might be helpful to include a reference -- up to you in the end!

Reply:
good idea, i'll add this to a follow up issue (I want to add references to all types, which will be a bit of work)